src/enigma2/Admin.cpp

```diff
@@ -159,6 +161,43 @@ bool Admin::LoadDeviceInfo()
 
   m_deviceInfo = DeviceInfo(serverName, enigmaVersion, imageVersion, distroVersion, webIfVersion, webIfVersionAsNum);
 
+  hRoot=TiXmlHandle(pElem);
+
+  TiXmlElement* pNode = hRoot.FirstChildElement("e2frontends").Element();
```

Comment:
`const TiXmlElement*`?

Reply:
As it's used down further for the tunerNode it can't be const.

Reply:
Really? I cannot see any re-assigment for `pNode`. `tunerNode`is another var which cannot be `const` as it will be re-assigned, yes.